src/dialog/_dialog.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "../variables";
+@import "../mixins";
+
+.mdl-dialog {
+    display: block;
+    @include dialog-width();
+    margin: auto;
+    background: white;
+    position: fixed;
+    top: 5%;
+    right: 0;
+    left: 0;
+    z-index: 11;
+    @include shadow-24dp();
+
+    @media (min-height: 475px) {
+      top: 35%;
+    }
+
+    &:not([open]) {
+        display:none;
+    }
+
+    &::backdrop{
+        background-color: $dialog-backdrop-color;
+    }
+
+    &-backdrop {
+        background-color: $dialog-backdrop-color;
+        position:fixed;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        z-index: 10;
+        height: 100%;
+        width: 100%;
+    }
+
+    &__title {
+        padding-left: 24px;
+        padding-right: 24px;
+        padding-top: 24px;
+    }
+    &__actions {
+        padding: 8px 8px 8px 24px;
+        display: flex;
+        flex-direction: row-reverse;
+        flex-wrap: wrap;
+        * {
+            margin-right: 8px;
+            height: 36px;
+            &:last-child {
+                margin-right: 0;
+            }
+        }
+        &--full-width {
+          padding: 0 0 8px 0;
+          * {
```

Comment:
> It is better to use :first-child than :last-child, since it is much more efficient. 

Do you have source for this? I have never heard this and I’m kind of skeptical.